### DISCUSSION TITLE
Open All Windows button

### DISCUSSION THREAD

**Discussion by MagmarFire** [RetroAchievements/RAIntegration#903] [Completed]
I think it would be a nice QOL addition if there were an option alongside the Asset Editor, RP Monitor, Memory Inspector, etc. that opened up all of those windows at the same time. My current workflow invariably involves opening up all of them one at a time, which can get somewhat annoying if some unfortunate circumstance, such as a crash resulting from experimenting with RAM digging, forces me to open them all up again manually.

**Comment by televandalist** [comment]
![1](https://user-images.githubusercontent.com/45054151/178882380-08ea20c6-3ddc-4ba3-9289-2ede128b7f21.PNG)
I can dream.

**Comment by Jamiras** [comment]
Implemented in #912
